src/nxt_sprintf.c

```diff
@@ -554,6 +555,11 @@ nxt_vsprintf(u_char *buf, u_char *end, const char *fmt, va_list args)
 
     copy:
 
+        if (nxt_slow_path(p == NULL)) {
+            p = (u_char *) null;
```

Comment:
I don't like having to cast this.  I'll write a patch to allow using this without the cast.